vector/src/main/java/im/vector/app/features/home/room/detail/TimelineViewModel.kt

```diff
@@ -719,22 +719,12 @@ class TimelineViewModel @AssistedInject constructor(
             when (itemId) {
                 R.id.timeline_setting          -> true
                 R.id.invite                    -> state.canInvite
-<<<<<<< HEAD
-                R.id.open_matrix_apps          -> false // Tchap: there are no matrix apps
-                R.id.voice_call                -> BuildConfig.IS_VOIP_SUPPORTED && state.isWebRTCCallOptionAvailable() // Tchap: check if voip is enabled
-                R.id.video_call                -> BuildConfig.IS_VOIP_SUPPORTED && // Tchap: check if voip is enabled
-                        (state.isWebRTCCallOptionAvailable() || state.jitsiState.confId == null || state.jitsiState.hasJoined)
-                // Show Join conference button only if there is an active conf id not joined. Otherwise fallback to default video disabled. ^
-                R.id.join_conference           -> !state.isWebRTCCallOptionAvailable() && state.jitsiState.confId != null && !state.jitsiState.hasJoined
-                R.id.search                    -> true
-=======
                 R.id.open_matrix_apps          -> true
```

Comment:
Tchap is not recognized as a matrix app. Please restore the change coming from Tchap.